### DISCUSSION TITLE
Group data in BB Database by data type in Table.

### DISCUSSION THREAD

**Discussion by LTCTHITA** [Budibase/budibase#12224] [Help] [answered]
I want to segment data Then subgroup it further.
When the data is divided into subgroups Show data in that subgroup.
![Untitled](https://github.com/Budibase/budibase/assets/93240913/a95c6d1f-59e0-4158-bc3e-108698f19d2d)

Show information that has been sub-grouped. in each Row, but shows data for the entire selected group.
![Untitled1](https://github.com/Budibase/budibase/assets/93240913/e07083e9-f61f-4ecb-b558-d24be50222da)

**Comment by ConorWebb96** [comment, marked as answer]
Hey @LTCTHITA,

I don't believe the internal Budibase table has the ability to do this anymore. The groupBy functionality was removed quite a long time ago. You would be able to do this using custom queries using an external database.

This functionality is likely to return with the implementation of SQS, this won't be an immediately released along side it but will likely come whenever you are able to write custom queries for the internal table.

I hope this helps.
